### PR TITLE
Feat: const fixed size arrays

### PR DESCRIPTION
**Pull request by remybar**
See issue https://github.com/kkrt-labs/kakarot-ssj/issues/806.

The first commit uses const fixed-size arrays in the code while the second one uses const fixed-size arrays in tests.

## Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

- - -
This change is [Reviewable](https://reviewable.io/reviews/kkrt-labs/kakarot-ssj/822)

### CODE REVIEW

File: crates/contracts/src/account_contract.cairo [hunk 1]
```diff
@@ -81,6 +81,41 @@ pub mod AccountContract {
 
     const VERSION: u32 = 000_001_000;
 
+    const KAKAROT_VALIDATION_FAILED: [
```
Comment (enitrat) [inline crates/contracts/src/account_contract.cairo:84]: can you put this inside the `contracts::errors` module please

File: crates/evm/src/instructions/environmental_information.cairo [hunk 2]
```diff
@@ -649,7 +653,7 @@ mod tests {
 
     #[test]
     fn test_calldatacopy_basic() {
-        test_calldatacopy(32, 0, 3, array![4, 5, 6].span());
+        test_calldatacopy(32, 0, 3, ARRAY_3.span());
```
Comment (enitrat) [inline crates/evm/src/instructions/environmental_information.cairo:656]: can this not be inlined as 

```suggestion
        test_calldatacopy(32, 0, 3, [4,5,6].span());
```
?

If yes, do this for all the tests. If not, revert to the previous implementation, as we don't really about perf for tests.

**Review by enitrat — COMMENTED** [review]
can you also adapt `pow2` and `pow256_rev` ?

**Comment by remybar** [comment]
> can you also adapt `pow2` and `pow256_rev` ?

Yes, done 👍

**Review by enitrat — APPROVED** [review]

**Pull request merged**